frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -711,9 +715,10 @@ class TableInteractive extends Component {
     const isAscending = isSorted && sort[sortIndex][0] === "asc";
 
     return (
-      <Draggable
+      <TableDraggable
         /* needs to be index+name+counter so Draggable resets after each drag */
-        enableUserSelectHack={!isVirtual}
+        enableUserSelectHack={false}
```

Comment:
The default "hack" adds custom styles directly to the DOM and it doesn't work with CSP. Now we replicate this part with emotion.